docs/schema/documenttemplate-element-contenttype.md

```diff
@@ -72,7 +73,7 @@ The following sections describe attributes, child elements, and parent elements.
 </colgroup>
 <tbody>
 <tr class="odd">
-<td align="left"><p><span sdata="link"><a href="contenttype-element-contenttype.md">ContentType Element (ContentType)</a></span></p></td>
+<td align="left"><p><span><a href="contenttype-element-contenttype.md">ContentType Element (ContentType)</a></span></p></td>
```

Comment:
Same here, this needs to be turned into a Markdown table.  If we do any edits to an HTML table for disallowed HTML attributes, we need to rework the table, and turn it into a Markdown table.